### PR TITLE
style: different style to active button on current page

### PR DESCRIPTION
**Pull request by galarist**
Co-Authored-By: Lisa Li <61769853+2016lisali@users.noreply.github.com>

### CODE REVIEW

File: src/components/TopNav.tsx [hunk 1]
```diff
@@ -117,14 +137,28 @@ export default function TopNav() {
...
+        color: "text.primary",
+    },
     ListItemButton: {
+        color: "text.primary",
         textAlign: "center",
         justifyContent: "center",
         mb: 0.4,
     },
+    divider: {
+        marginLeft: 0.5,
```
Comment (2016lisali) [inline src/components/TopNav.tsx]: marginLeft and marginRight can be replaced with mx: 0.5 which is much simpler :)

**Review by 2016lisali — CHANGES_REQUESTED** [review]
Look good. Just 1 tiny part needs to be modified :)

**Review by 2016lisali — APPROVED** [review]
Good work, thanks, Chris.

**Review by QiujuYang — APPROVED** [review]

**Review by B08waffles — APPROVED** [review]
Looks good, I like how you've used the `...` operator to use two styles at once, well done!

**Pull request merged**